merkle/hstar2_test.go

```diff
@@ -224,6 +201,17 @@ func TestPaddedBytes(t *testing.T) {
 		}
 	}
 }
+
+// b64 converts a base64 string into []byte.
```

Comment:
would `fromB64` be better?  I was hoping to keep a short name because this function gets used a lot when forming table driven tests and a long name just pushes the test vectors off the screen.